bedrock/firefox/templates/firefox/browsers/quantum.html

```diff
@@ -0,0 +1,55 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/browsers/quantum" %}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ ftl('page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('page-desc') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('quantum') }}
+{% endblock %}
+
+{% block content %}
+
+{% call hero(
+  title=ftl('the-latest-firefox'),
+  class='mzp-has-image',
+  image_url='img/firefox/new/desktop/ie9/browser.jpg',
```

Comment:
Any reason why this image path is `/new/desktop/ie9/`? Unless it's specific for that browser, suggest putting it in `/img/firefox/browsers/quantum/` so it's more easily associated with this page?